pkg/apis/crd/v1alpha1/types.go

```diff
@@ -291,3 +291,37 @@ type TLSProtocol struct {
 	// SNI (Server Name Indication) indicates the server domain name in the TLS/SSL hello message.
 	SNI string `json:"sni,omitempty"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeLatencyMonitor is used to monitor the latency between nodes in a Kubernetes cluster. It is a singleton resource,
+// meaning only one instance of it can exist in the cluster.
+type NodeLatencyMonitor struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec NodeLatencyMonitorSpec `json:"spec"`
+}
+
+type NodeLatencyMonitorSpec struct {
+	// PingInterval specifies the interval in seconds between ping requests.
+	// Ping interval should be greater than or equal to 1s(one second).
```

Comment:
there should be a space before `(one second)`, but I think we could just remove `(one second)`.